content/plugins/fabric.md

```diff
@@ -153,6 +159,7 @@ node_templates:
             # Path to the script relative to the blueprint directory
             script_path: scripts/start.sh
             MY_ENV_VAR: some-value
+            use_sudo: false
```

Comment:
the basic example only specified the required inputs (+ some env var),
consider removing this and adding an example where this is set to true so that people don't confuse this for a required property (I know it is documented that it defaults to `false` but you know how it is with documentation. copy/paste is much more likely than reading the docs